python/k4FWCore/IOSvc.py

```diff
@@ -33,10 +33,10 @@ def __setattr__(self, attr, value):
             return
 
         # Allow to specify a single string for input when what we want is a list
-        if attr == "input":
+        if attr == "input" or attr == "Input":
```

Comment:
But making a tuple is significantly slower for just a few characters less:
``` python
import timeit

def find(value):
    if value == 1 or value == 2:
        return
    return
def findtuple(value):
    if value in (1, 2):
        return
    return

time1 = timeit.timeit('find(1)', globals=globals(), number=100000000)
time2 = timeit.timeit('findtuple(1)', globals=globals(), number=100000000)
time3 = timeit.timeit('find(2)', globals=globals(), number=100000000)
time4 = timeit.timeit('findtuple(2)', globals=globals(), number=100000000)
time5 = timeit.timeit('find(3)', globals=globals(), number=100000000)
time6 = timeit.timeit('findtuple(3)', globals=globals(), number=100000000)
print(time1, time2, time3, time4, time5, time6)
```
I get
```
3.1558683200273663 3.1746458479901776 3.558305319980718 3.810415144020226 3.5353208070155233 4.249286936013959
```

Reply:
But it is more readable and extensible, and this doesn't happen in a tight loop. And your example is not using strings.

But we should put "Input" first, because that should be faster and the first if should shortcut...